Flabbergasted-Flamingos/bangazon-api
# Jdb faker

# Description
Adds the django-seed app to api to allow us to seed faker data in the project from the command line or in the code.  

## Number of Fixes
1

## Related Ticket(s)
closes #35 

## Problem to Solve
We can use this functionality to seed data after remigrating.  

## Steps to Test Solution

Make sure you install django-seed in your virtual enviroment: 
```
pip install django-seed
```
then pull the branch and remigrate:
```
git fetch --all
git checkout JDB_faker
rm -rf api/migrations
rm db.sqlite3
python manage.py makemigrations api
python manage.py migrate
```
now seed data in the database from the command line:
```
python manage.py seed api --number=15
```

This should add fifteen resources from each model to the database.  
Check out https://pypi.org/project/django-seed/ for more options for seeding data.
